src/courseware/course/Course.jsx

```diff
@@ -76,13 +77,17 @@ const Course = ({
         <title>{`${pageTitleBreadCrumbs.join(' | ')} | ${getConfig().SITE_NAME}`}</title>
       </Helmet>
       <div className="position-relative d-flex align-items-center mb-4 mt-1">
-        <CourseBreadcrumbs
-          courseId={courseId}
-          sectionId={section ? section.id : null}
-          sequenceId={sequenceId}
-          isStaff={isStaff}
-          unitId={unitId}
-        />
+        {!navigationDisabled && (
```

Comment:
```suggestion
        {navigationDisabled || (
```

Dual negatives are not great for reading code.

Reply:
Yes, of course. Thanks!